src/test/java/com/exasol/adapter/dialects/db2/DB2SqlDialectIT.java

```diff
@@ -0,0 +1,233 @@
+import static com.exasol.dbbuilder.dialects.exasol.AdapterScript.Language.JAVA;
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static com.exasol.matcher.TypeMatchMode.NO_JAVA_TYPE_CHECK;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.*;
+import org.testcontainers.containers.Db2Container;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.dbbuilder.dialects.DatabaseObject;
+import com.exasol.dbbuilder.dialects.exasol.*;
+
+@Tag("integration")
+@Testcontainers
+class DB2SqlDialectIT {
+    private static final String SOURCE_SCHEMA = "DB2INST1";
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>(
+            EXASOL_DOCKER_REFERENCE).withReuse(true);
+    @Container
+    private static final Db2Container DB2 = new Db2Container(EDB2_DOCKER_REFERENCE);
+    private static Connection exasolConnection;
+    private static Connection db2Connection;
+    private static ExasolObjectFactory objectFactory;
+    private static ExasolSchema adapterSchema;
+    private static AdapterScript adapterScript;
+    private static ConnectionDefinition jdbcConnectionDefinition;
+
+    @BeforeAll
+    static void beforeAll() throws BucketAccessException, InterruptedException, TimeoutException,
+            JdbcDatabaseContainer.NoDriverFoundException, SQLException {
+        exasolConnection = EXASOL.createConnection("");
+        db2Connection = DB2.createConnection("");
+        objectFactory = new ExasolObjectFactory(exasolConnection);
+        adapterSchema = objectFactory.createSchema("ADAPTER_SCHEMA");
+        uploadDriverToBucket();
+        adapterScript = installVirtualSchemaAdapter(adapterSchema);
+        jdbcConnectionDefinition = createAdapterConnectionDefinition();
+    }
+
+    private static AdapterScript installVirtualSchemaAdapter(final ExasolSchema adapterSchema)
+            throws InterruptedException, BucketAccessException, TimeoutException {
+        final Bucket bucket = EXASOL.getDefaultBucket();
+        bucket.uploadFile(PATH_TO_VIRTUAL_SCHEMAS_JAR, VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+        final String content = "%scriptclass com.exasol.adapter.RequestDispatcher;\n" //
+                + "%jar /buckets/bfsdefault/default/" + VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION + ";\n" //
+                + "%jar /buckets/bfsdefault/default/drivers/jdbc/" + JDBC_DRIVER_NAME + ";\n";
+        return adapterSchema.createAdapterScript("EXASOL_ADAPTER", JAVA, content);
```

Comment:
if you update the test-db-builder-java dependency you can do 
```java
adapterSchema.createAdapterScriptBuilder("EXASOL_ADAPTER").language(JAVA)
  .bucketFsContent("com.exasol.adapter.RequestDispatcher", "/buckets/bfsdefault/default/" + VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION, "/buckets/bfsdefault/default/drivers/jdbc/" + JDBC_DRIVER_NAME);
```

Reply:
I think the current way is more readable.

Reply:
I disagree. You can also format the builder call or extract a constant. 
In addition, using the builder call we stay on the correct abstraction level. 
In case the CREATE VIRTUAL SCHEMA command syntax would change we only need to update test-db-builder.

Reply:
bucketFsContent() method only allows me to send one jar as an argument. But I have two jars, is it supported?

Reply:
yes. You need to update the version